requirements.txt

```diff
@@ -1,4 +1,4 @@
 ocrd >= 2.13.1
```

Comment:
2.16.2 has the fix for empty reading order

Reply:
thx, update is coming...